static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -143,7 +136,7 @@ interface State {
   title: string;
   userHasModified: boolean;
   errors?: Record<string, any>;
-  selectedDashboard?: SelectValue<string>;
+  selectedDashboard?: string;
```

Comment:
2 small things:
can we remove `dashboards` from state then?
nit: can we type this `selectedDashboard?: DashboardDetail['id'];` or whatever
because I saw this as `string` and couldn't immediately tell if it was dashboard name or id